zeebe/qa/update-tests/src/test/java/io/camunda/zeebe/test/RollingUpdateTest.java

```diff
@@ -53,6 +55,7 @@
  * <p>The important part is that we should be aware whether rolling update is possible between
  * versions.
  */
+@Execution(ExecutionMode.CONCURRENT)
```

Comment:
When I tried that last it was causing too much flakiness, but that was a long time ago :sweat_smile:

Reply:
Nah, still the same, I just forgot to close the PR :disappointed: